src/main/java/school/faang/streamapi1/StreamApiHandler.java

```diff
@@ -0,0 +1,27 @@
+package school.faang.streamapi1;
+
+import java.util.List;
+
+public class StreamApiHandler {
+    public static int getSum(List<Integer> numbers) {
+        return numbers.stream().filter(num -> num % 2 == 0).mapToInt(num -> num).sum();
+    }
+
+    public static int maxElem(List<Integer> numbers) {
+        if (numbers.isEmpty()) return 0;
```

Comment:
в таком стиле писать код - плохая затея, не совсем читабельно. Лучше использовать стандратный подход:
```suggestion
        if (numbers.isEmpty()) {
             return 0;
        }
```